src/installLogsPrinter.js

```diff
@@ -72,13 +75,20 @@ function installLogsPrinter(on, options = {}) {
         allMessages[data.spec][data.test] = messages;
       }
 
-      logToTerminal(messages, options);
+      let printL2C = options.printLogsToConsole || "onFail";
```

Comment:
Not here. Right on the first line of of the installLogsPrinter. Before the validation. Also the schema should not allow for undefined.

Reply:
`options.printLogsToConsole = options.printLogsToConsole || "onFail"`